### ISSUE TITLE
Unsolved style/colorscheme bits

### ISSUE THREAD

**Issue by bitprophet** [bitprophet/alabaster#6]
tl;dr let's make all the coloring configurable, right now only ~30-50% of the common elements have configurable color options.

* [ ] 'See also' blocks are yellow background & font sizes feel off. This element does not appear at all in Requests' docs so suspect this styling is inherited from the basic theme or something. Fix it up.
* [x] Similarly, 'warning' blocks have zero color styling, they should be a controllable color that defaults to a pleasing shade of red.
    * This is especially important as RTD inserts a warning block (a `div class='admonition warning'`) for 'on an old version' notices.
* [ ] Note blocks are currently one of the gray colors; see if they might want to be something else?
* [ ] `dt:target, .highlight` rule is light-colored but idk what it actually maps to offhand.
* [ ] `hr` is colored but is not used often in (my) Sphinx docs, so it's been left alone for now
* [ ] Body background is hardcoded to be white right now, might as well make it an option sometime
* [ ] `div.admonition tt.xref, div.admonition a tt` - not entirely sure what these are offhand
    * DItto top-level `tt`
* [ ] `div.topic` - also not sure where this appears, topics are usually header tags?
* [ ] `img.screenshot`, `table.docutils` (+ sub-items), ???

**Comment by SamWhited** [comment]
I'd also love to see `pre_border` added to this list (right now you can only set `pre_bg`).

**Comment by bitprophet** [comment]
This should pretty much all be fixed in #84, any other similar changes should become their own tickets. ![](http://i.somethingawful.com/forumsystem/emoticons/emot-toot.gif)
